rental.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'book'
+require_relative 'person'
+
+class Rental
+  def initialize(date, book, person)
+    @person = person
+    @book = book
+    @date = date
+    book.rental << self
+    person.rental << self
```

Comment:
- [ ] Hello! 👋 Just a friendly reminder: in both the Person and Book classes, we've declared @rentals instance variables. To align with a clearer and more explicit naming convention, let's make the adjustment to book.rentals and person.rentals. 📚💻 This enhances code readability and distinguishes the rentals associated with books and people. 🌟 Let's keep our codebase consistent and straightforward! 🛠️✨

```suggestion
   book.rentals << self
   person.rentals << self
```